test/views/declaration/ConsignmentReferencesViewSpec.scala

```diff
@@ -71,14 +71,14 @@ class ConsignmentReferencesViewSpec extends UnitViewSpec with CommonMessages wit
 
       "display section header" in {
 
-        createView().getElementById("section-header").text() must include(messages("supplementary.consignmentReferences.heading"))
+        createView().getElementById("section-header").text() must include(messages("declaration.section.1"))
       }
 
       "display empty input with label for DUCR" in {
 
         val view = createView()
 
-        view.getElementById("ducr_ducr-label").text() mustBe messages("supplementary.consignmentReferences.ducr.info")
+        view.getElementsByAttributeValue("for", "ducr_ducr").text() mustBe messages("supplementary.consignmentReferences.ducr.info")
```

Comment:
Have you considered finding elements by class?

Reply:
This is specifically testing accessibility i.e. its looking for the element that is the label for a named field.     The pattern is used elsewhere and in movements